# DP: Make sure libc_nonshared.a is compiled with -fPIC on m68k

--- ports/sysdeps/m68k/Makefile	2006-06-01 11:20:48.000000000 +0200
+++ ports/sysdeps/m68k/Makefile	2006-06-01 11:37:21.000000000 +0200
@@ -44,3 +44,7 @@
 shared-only-routines += libgcc-compat
 endif
 endif
+
+# Build objects in libc_nonshared.a with -fPIC (instead of -fpic) to avoid
+# possible linkage problems.
+CFLAGS-.oS += -fPIC
